CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing to Command Set Registry
+
+Thanks for taking the time to contribute. We're thankful and will do our best to make progress with you.
+
+> In case this your first time contributing on GitHub, you can watch this [series](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) to learn how to contribute.
+
+We have a [Code of Conduct](CODE_OF_CONDUCT), please take a look before you proceed.
+
+## Contributing to docs
+
+If you found a typo, just make a pull request with the fix. But if you're planning to write docs for a command set or a command, raise an issue and then proceed.
+
+## Requesting for a new feature/command
+
+Raise an issue with a brief description about your problem and in what way a command/feature can solve it.
+
+## Making changes to existing commands
+
+First, raise an issue describing your change. After discussion, you can make a pull request.
+
+## Creating a new command set
+
+Let's create a small `greetings` command set to understand the flow.
+
+### 1. Fork this repo & clone it.
+
+```sh
+$ git clone https://github.com/<username>/command-sets
+```
+
+### 2. Create a new directory named `greetings` inside the `command-sets` repository.
+
+```sh
+$ cd command-sets && md greetings
+```
+
+### 3. Create two new files inside `greetings` to greet in Spanish & French.
+
+```sh
+$ cd greetings && touch {hola,bonjour}.js
```

Comment:
here we should adopt the Nimbella project format as this is the long term direction. So `mkdir -p greetings/packages/default && cd ... && touch ...`

Reply:
I guess it would be better to advocate it once we have `command-sets` ported to the Nimbella project format. What would you say?

Reply:
It's compatible already, that is if you use `packages/default/hola.js` instead of `hola.js` in the yml file, the command set install will still work - so it's a forward compatible change.

Reply:
Got it.